### DISCUSSION TITLE
How to use -ssw (Compress files open for writing) switch?

### DISCUSSION THREAD

**Discussion by shankun** [rikyoz/bit7z#99] [Q&A] [answered]
When I use 7z in command line, there is a "-ssw" (Compress files open for writing) switch,
Compresses files open for writing by another applications. If this switch is not set, 7-zip doesn't include such files to archive.

Example
7z a archive.7z -ssw *.txt

compresses all *.txt files in current folder including files open for writing by another applications.
Wonder how I can use this feature in this library.

**Comment by rikyoz** [comment, marked as answer]
Hi!
> Wonder how I can use this feature in this library.

Unfortunately, this feature is not supported by the current version of bit7z.

However, adding it shouldn't be a problem; hopefully, I'll be able to do it to the next stable version.
